package.json

```diff
@@ -99,6 +99,8 @@
     "@leafygreen-ui/tooltip": "9.1.6",
     "@leafygreen-ui/typography": "16.4.1",
     "@rjsf/core": "4.2.0",
+    "@sentry/react": "7.56.0",
+    "@sentry/types": "7.56.0",
     "ansi_up": "5.1.0",
     "antd": "4.20.0",
```

Comment:
I'm starting to see a ton of build warnings when building the app. 
![image](https://github.com/evergreen-ci/spruce/assets/4605522/fbc6c046-1101-41fd-b9f0-c658a1bf0f68)